app/plugin.go

```diff
@@ -101,6 +102,23 @@ func (a *App) InitPlugins(pluginDir, webappPluginDir string) {
 		a.Plugins = env
 	}
 
+	prepackagedPluginsDir := "./prepackaged_plugins"
```

Comment:
This will always be relative to the binary? Or will it stop working if someone is running the binary from the non-standard location?

Reply:
Yep, you're absolutely right. Fixed.